### ISSUE TITLE
jquery-integration optional feature won't enable

### ISSUE THREAD

**Issue by chrism** [emberjs/ember.js#16723]
With the latest version of canary my console is logging

`DEPRECATION: You have disabled the 'jquery-integration' optional feature. You now have to delete '@ember/jquery' from your package.json`

Following the **Opting into jQuery** section of the deprecation guide
https://emberjs.com/deprecations/v3.x/#toc_jquery-event

It recommends to 

```bash
ember install @ember/jquery
ember install @ember/optional-features
ember feature:enable jquery-integration
```

both those packages are installed

```json
"@ember/jquery": "^0.5.1",
"@ember/optional-features": "^0.6.1",
```

and running the enable feature command seems to work (logs correctly and adds the feature to config/optional-features.json)

```
⇒  ember feature:enable jquery-integration
Enabled jquery-integration. Be sure to commit config/optional-features.json to source control!
```

However, running the server still displays the deprecation.

```
⇒  EMBER_CLI_MODULE_UNIFICATION=true ember serve
DEPRECATION: You have disabled the `jquery-integration` optional feature. You now have to delete `@ember/jquery` from your package.json
```

If I follow the instructions and remove the `@ember/jquery` package it successfully removes the deprecation warning.

But if I then re-add the package though, and set the `jquery-integration` optional feature to true the warning returns.

**Comment by rwjblue** [comment]
Awesome, thanks for the write up! Would you mind throwing together a repo that demonstrates the issue?

**Comment by chrism** [comment]
Sure thing :)

Is it OK on a branch?

I've just pushed it here if so
https://github.com/chrism/emberjs-2018/tree/jquery-integration-issue

if you run
`EMBER_CLI_MODULE_UNIFICATION=true ember serve`
you'll see the deprecation, but you can see

config/optional-features.json
```json
{
  "application-template-wrapper": false,
  "jquery-integration": true,
  "template-only-glimmer-components": true
}
```

& `@ember/jquery` in package.json

Let me know if there is anything else I can help with.

**Comment by rwjblue** [comment]
Thanks for being so thorough in the writeup here! It really made tracking down the issue much easier.

Problem should be resolved with `@ember/jquery@0.5.2`...

**Comment by chrism** [comment]
You're welcome!

Thanks :)
